src/seedu/addressbook/data/person/Email.java

```diff
@@ -6,8 +6,9 @@
  * Represents a Person's email in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidEmail(String)}
  */
-public class Email {
+public class Email implements Printable {
 
+    public static final String CATEGORY = "Email: ";
```

Comment:
:+1:  for extracting this as a constant string rather than use individual words in the code :-)